output.go

```diff
@@ -0,0 +1,14 @@
+package shaping
+
+import "image"
+
+type Output interface {
```

Comment:
I think this needs to return baselineing information in addition to bounds.

Reply:
Good point. I think harfbuzz puts the dot on the origin, so the baseline is assumed to exactly follow the x-axis (ie. y=0) and the bounds could extend below but I could be wrong. Either way I agree it would be good to be explicit 👍